src/s_conf.c

```diff
@@ -7523,10 +7556,6 @@ int	_test_set(ConfigFile *conf, ConfigEntry *ce)
 				}
 			}
 		}
-		else if (!strcmp(cep->ce_varname, "anti-spam-quit-message-time")) {
```

Comment:
I think we should should keep old option available so people who like current functionality don't have to change config.